### PR TITLE
fix: shuffle batches

### PR DESCRIPTION
**Pull request by florian-hoenicke**
Before:
image

After:
image

### CODE REVIEW

File: finetuner/tuner/dataset/samplers.py [hunk 1]
```diff
@@ -150,6 +151,7 @@ def _prepare_batches(self):
             batches.append(batch)
 
         self._batches = batches
+        random.shuffle(self._batches)
```
Comment (JoanFM) [inline finetuner/tuner/dataset/samplers.py:154]: there should be some option right? sometimes to debug is nice to disable?

File: finetuner/tuner/dataset/samplers.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+import random
```
Comment (tadejsv) [inline finetuner/tuner/dataset/samplers.py:1]: `shuffle` is already imported

**Review by tadejsv — CHANGES_REQUESTED** [review]

File: finetuner/tuner/dataset/samplers.py (hunk 2 above)
Comment (florian-hoenicke) [reply finetuner/tuner/dataset/samplers.py]: thanks

File: finetuner/tuner/dataset/samplers.py (hunk 1 above)
Comment (florian-hoenicke) [reply finetuner/tuner/dataset/samplers.py]: I don't think so. There will be no advantage of having shuffling disabled

**Review by tadejsv — APPROVED** [review]

**Pull request merged**